crates/polars-plan/src/dsl/python_udf.rs

```diff
@@ -26,88 +21,10 @@ pub static mut CALL_COLUMNS_UDF_PYTHON: Option<
 pub static mut CALL_DF_UDF_PYTHON: Option<
     fn(s: DataFrame, lambda: &PyObject) -> PolarsResult<DataFrame>,
 > = None;
-#[cfg(feature = "serde")]
-pub(super) const MAGIC_BYTE_MARK: &[u8] = "PLPYUDF".as_bytes();
-static PYTHON_VERSION_MINOR: Lazy<u8> = Lazy::new(get_python_minor_version);
 
-#[derive(Debug)]
-pub struct PythonFunction(pub PyObject);
```

Comment:
I have moved `PythonFunction` from `polars-plan/../python_udf.rs` to `polars-utils` for re-use